test/unit/ngsiv2/contextRequests/singleMeasuresRawTypes.json

```diff
@@ -1,12 +1,6 @@
 {
   "humidity":{
     "type": "degrees",
-     "value": { 
-         "type": "Buffer",
-         "data": [
-             51,
-             50
-         ]
-     }
+     "value": "32"
```

Comment:
This is not right, the expected result for the request is "3332", which is an string with the hex representation of the data sent ( a string, "32")

Reply:
Here the part of the test that send the request:
https://github.com/telefonicaid/iotagent-json/blob/36bcc64fee817c4289f048b9ded3b2ea6fe5f4d9/test/unit/ngsiv2/HTTP_receive_measures-test3.js#L190-L205